notebooks/SMAP/smap_rasterio_issues.ipynb

```diff
@@ -0,0 +1,2339 @@
+{
```

Comment:
<p>After adding the file to my smap_data directory, I still get this error:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">OSError: [Errno -101] NetCDF: HDF error: '/home/jovyan/other-repos/NSIDC-Data-Tutorials/notebooks/SMAP/smap_data/SMAP_L4_SM_gph_20150331T013000_Vv7032_001.h5'
</pre><p> </p><p> </p><p> </p><p>I'm using xarray v2024.11.0 and rioxarray v0.17.0</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>